qiskit/algorithms/gradients/spsa_sampler_gradient.py

```diff
@@ -0,0 +1,103 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Gradient of Sampler with Finite difference method."""
+
+from __future__ import annotations
+
+from typing import Sequence
+
+import numpy as np
+
+from qiskit.circuit import Parameter, QuantumCircuit
+from qiskit.primitives import BaseSampler
+
+from .base_sampler_gradient import BaseSamplerGradient
+from .sampler_gradient_result import SamplerGradientResult
+
+
+class SPSASamplerGradient(BaseSamplerGradient):
+    """
+    Compute the gradients of the sampling probability by the Simultaneous Perturbation Stochastic
+    Approximation (SPSA).
+    """
+
+    def __init__(
+        self,
+        sampler: BaseSampler,
+        epsilon: float = 1e-6,
```

Comment:
Is it better to remove the default value of `epsilon` for SPSA as well as finite diff?